webapp/src/Entity/Problem.php

```diff
@@ -390,7 +390,7 @@ public function removeContestProblem(ContestProblem $contestProblem): void
     /**
      * @return Collection|ContestProblem[]
      */
-    public function getContestProblems()
+    public function getContestProblems(): \Doctrine\Common\Collections\Collection|array
```

Comment:
Can we have the shortform here? Is there not an automation for this?

Reply:
There is in php cs fixer